src/restore.c

```diff
@@ -153,13 +153,55 @@ restore_main(int32_t argc, char **argv)
 	}
 
 	uint32_t line_no;
+	as_vector directories;
+	as_vector_init(&directories, sizeof(char*), 1);
+
+	// restoring from multiple directories
+	if (conf.directory_list != NULL) {
+
+		char *dir_clone = safe_strdup(conf.directory_list);
+		split_string(dir_clone, ',', false, &directories);
+
+		for (uint32_t i = 0; i < directories.size; i++) {
+			char *dir = as_vector_get_ptr(&directories, i);
+
+			if (conf.parent_directory) {
+				size_t path_size = strlen(conf.parent_directory) + strlen(dir) + 1;
+				char *tmp_dir = dir;
+				dir = cf_malloc(path_size);
+				snprintf(dir, path_size, "%s%s", conf.parent_directory, tmp_dir);
+			}
+
+			if (!get_backup_files(dir, &status.file_vec)) {
+				err("Error while getting backup files from directory_list entry: %u", i);
```

Comment:
Whether you pick to make the `/` optional or not this should probably change so they know if they formatted it wrong. This should probably go for other error messages as well where asbackup fails because of an incorrect path or something
```suggestion
				err("Error while getting backup files from directory_list entry: %s", dir);
```